README.md

```diff
@@ -10,9 +10,13 @@ SYNAPSE_OAUTH_CLIENT_SECRET=xxxxxx
 TEAM_TO_ROLE_ARN_MAP=[{"teamId":"xxxxxx","roleArn":"arn:aws:iam::xxxxxx:role/ServiceCatalogEndusers"}, ...]
 AWS_REGION=us-east-1
 SESSION_TIMEOUT_SECONDS=43200
+USER_CLAIMS=userid
 ```
 Note: When mapping team ID to AWS Role, this app' uses the first match it encounters,
-iterating through the team/role list in the order given.  
+iterating through the team/role list in the order given. 
+
+Note:  `USER_CLAIMS` is a comma separated lst of claims from the list of availalble cliams, given here:
```

Comment:
lst/list 

I don't quite understand the purpose of supplying multiple claim values.  could you please enlighten me?

Reply:
Yes, certainly. The feature was requested by @tthyer  in [SC-85](https://sagebionetworks.jira.com/browse/SC-85)

> Proposed format is to use the current format but append the email address, separated from the id by a delimiter character not allowed in email addresses, for example, "ServiceCatalogEndusers/3388489:tess.thyer@gmail.com". This is assuming that it is best to keep the Synapse ID in the string rather than simply using the email as the unique identifier.

Reply:
ahh ok.  I thought she only requested `ServiceCatalogEndusers/tess.thyer@gmail.com` but this is probably better since email can change in synapse.  thanks.